client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEdit.tsx

```diff
@@ -77,7 +85,7 @@ export const RequestLineEdit = ({
             label={t('label.stock-on-hand')}
             sx={{ marginBottom: 1 }}
           />
-          {isProgram && useConsumptionData && (
+          {isProgram && extraFields && (
```

Comment:
These were using the wrong preferences... Values should be calculated in backend based on consumption preference